src/test/v4.test.ts

```diff
@@ -48,6 +49,26 @@ describe('v4', () => {
     assert.ok(id1 !== id2);
   });
 
+  test('should uses native randomUUID() if no option is passed', () => {
+    mock.method(native, 'randomUUID');
+
+    assert.equal((native.randomUUID as any).mock.callCount(), 0);
+    v4();
+    assert.equal((native.randomUUID as any).mock.callCount(), 1);
+
+    mock.restoreAll();
```

Comment:
Cool! I wasn't aware of the `mock` feature of Node's test runner until now.  Excellent change!

BTW, it looks like `mock.method()` return value is the mocked function with the proper TS "`Mock`" type, so we can capture that to avoid having to cast to the `any` type (which is always a bit of a code smell).

```suggestion
    const mockedV4 = mock.method(native, 'randomUUID');

    assert.equal(mockedV4.mock.callCount(), 0);
    v4();
    assert.equal(mockedV4.mock.callCount(), 1);

    mock.restoreAll();
```